individual-files/facilities.c

```diff
@@ -0,0 +1,28 @@
+#include <stdio.h>
+#include <string.h>
+
+char *sequence = "class department college office field library";
+void unique( const char *list, char *buf )
+{
+  char name[100];
+  int k;
+  buf[0] = 0;
```

Comment:
Erroneous Syntax (ES): This is not how you initialize an array

Rank 1 - Must Fix Now